src/main/java/it/pagopa/pn/datavault/PnDataVaultApplication.java

```diff
@@ -1,19 +1,21 @@
 package it.pagopa.pn.datavault;
 
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @SpringBootApplication
+@Slf4j
 public class PnDataVaultApplication {
 
 	public static void main(String[] args) {
 		// impostazione cache ttl
 		java.security.Security.setProperty("networkaddress.cache.ttl" , "1");
 		java.security.Security.setProperty("networkaddress.cache.negative.ttl", "1");
-		System.out.println("setted network cache");
+		log.debug("set network cache");
 		SpringApplication.run(PnDataVaultApplication.class, args);
```

Comment:
qui avevo messo system out volutamente perchè è proprio allo startup dell'applicativo e non sapevo se i log già c'erano. 
cmq vabbè se compaiono a posto per me

Reply:
In effetti non veniva loggato come avevo fatto. Spostando però il log dopo SpringApplication.run, la loggatura viene stampata correttamente. Se non va bene così, torno al stdout.